apps/person-api/src/main/java/no/nav/registre/testnorge/person/provider/PersonController.java

```diff
@@ -21,6 +26,25 @@
     @PostMapping
     public ResponseEntity<?> createPerson(@RequestBody PersonDTO personDTO) {
         service.createPerson(new Person(personDTO));
-        return ResponseEntity.ok().build();
+
+        URI uri = ServletUriComponentsBuilder
+                .fromCurrentRequest()
+                .path("/{ident}")
+                .buildAndExpand(personDTO.getIdent())
+                .toUri();
+        return ResponseEntity.created(uri).build();
+    }
+
+    @GetMapping("/{ident}")
+    public ResponseEntity<PersonDTO> getPerson(@PathVariable String ident) {
+        if (ident.length() != 11) {
```

Comment:
Hvis du skal vaidering brukt https://www.baeldung.com/spring-validate-requestparam-pathvariable Skal finnes en egen for regex

Reply:
Der var det en som var brukbar, ja 😄 Takk!